### ISSUE TITLE
KV Store Replication to Indexer

### ISSUE THREAD

**Issue by jdeer0618** [LukeMurphey/lookup-editor#18]
The default behavior for kvstores is not to replicate down to the indexers. However, Lookup Editor enables this replication by default. Is it possible to change Lookup Editor's behavior to not replicate to indexers or add an option to disable this in the UI?

**Comment by LukeMurphey** [comment]
I can do that. I'm trying to understand the use-case though. Could explain a bit?

Also, do you want this on a per-lookup basis or a global setting?

**Comment by LukeMurphey** [comment]
Oh wait. I think I understand now.

I bet you are talking about the "replicate = true" that the app always writes out for KV collections.

**Comment by jdeer0618** [comment]
Yeah, that’s it. Kvstores tend to get big and it is a pain in the ass to change the setting afterward in a SHC.

Best Regards,

James A. Deer Jr. (J.D.)

205.789.6025 - Cell*
866.601.4374 - Fax

"A good plan violently executed right now is far better than a perfect plan executed next week." -- George S. Patton

**Comment by MattUebel** [comment]
This is an issue for me as well. We'd like to default to not replicate to indexers, or at least be able to control this behavior.
